bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/submit/SubmitStatus.scala

```diff
@@ -41,12 +41,16 @@ object SubmitStatus {
 
   def submissionClosed(destination: Option[SubmitDestination]) = s"The ${destinationName(destination)} proposal server is currently closed. Please see the Gemini web pages for proposal submission dates"
 
-  private val ErrorRegex = """.* (\d\d\d\d.) ([\d\.]*).*""".r // Extract semester and version from the backend error message
+  private val VersionMismatchRegex = """.* (\d\d\d\d)([A|B]) \d{4}(\.[\d\.]*).*""".r // Extract semester and version from the backend error message
+
+  def parseServiceError405(destination: Option[SubmitDestination], message: String) = {
+    val VersionMismatchRegex(year, sem, version) = message
```

Comment:
Good point, this assume the backend is well behaved, which is not a 100% solid assumption